src/v2/Apps/Authentication/Utils/helpers.ts

```diff
@@ -103,7 +103,13 @@ export const handleSubmit = async (
 
       const result = await apiAuthWithRedirectUrl(res, afterAuthURL)
 
-      window.location.assign(result.href)
+      if (result.origin === sd.APP_URL) {
```

Comment:
If you're trying to redirect to an internal URL, we allow that. Otherwise, you get sent to the server.

Reply:
`getENV('APP_URL')`